types/voice-activity-detection/voice-activity-detection-tests.ts

```diff
@@ -0,0 +1,47 @@
+import vad from 'voice-activity-detection';
```

Comment:
Use the commonjs syntax `import vad = require('voice-activity-detection')`.

Consumers of the package who want to use `import vad from 'voice-activity-detection'` will need to add `"esModuleInterop": true` (which is the default in new tsconfigs), but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.